templates/partials/summary/collapsible-summary.html

```diff
@@ -2,6 +2,10 @@
 {% from "components/summary/_macro.njk" import onsSummary %}
 
 {%- set itemList = [] -%}
+{%- set tableHeader = ["Question", "Answer given"] -%}
```

Comment:
This is duplicated in the other template below, we could avoid adding the logic to the template (and repetition) by changing relevant context (eg. what `SummaryContext` class returns). `SectionSummaryContext` and `CalculatedSummaryContext` answers are always editable so we don't need to do the check.